tox.ini

```diff
@@ -75,7 +75,7 @@ skip_install = true
 changedir = doc/scapy
 deps = sphinx
 commands =
-  sphinx-apidoc -f --no-toc -d 1 --separate --module-first --templatedir=_templates --output-dir api ../../scapy ../../scapy/modules/ ../../scapy/libs/ ../../scapy/tools/ ../../scapy/arch/ ../../scapy/contrib/cansocket* ../../scapy/contrib/scada/* ../../scapy/all.py ../../scapy/layers/all.py ../../scapy/compat.py
```

Comment:
I remember sphinx failing on import. Did you fix that?

Reply:
Line 95 fixes the import. I've tested this locally.